client/src/Components/SearchBar.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import * as Style from './Searchbar.styles';
+import { weatherAPI } from '../API/api';
+import endpoints from '../Utils/endpoints';
+import { minCharacters } from '../Utils/minChars';
+import MultiWeather from './MultiCard';
+
+export default function SearchBar() {
+  const [inputValue, setInputValue] = useState('');
+  const [results, setResults] = useState([]);
+  const [chooseCityForecast, setChooseCityForecast] = useState([]);
+
+  useEffect(() => {
+    let timerID
+
+    if (inputValue.length > minCharacters) {
+      timerID = setTimeout(async () => {
+        try {
+          const res = await weatherAPI.get(endpoints.GET_CITY(inputValue));
+          setResults(res.data.locations);
+        } catch (error) {
+          alert(error);
+        }
+      }
+        , 900)
```

Comment:
still see incorrect formatting, please use npm scripts and packages provided in your project folder

Reply:
Ran code formatter, should be now properly formatted